de.dlr.sc.virsat.model.extension.requirements/src/de/dlr/sc/virsat/model/extension/requirements/csv/CsvFileReader.java

```diff
@@ -0,0 +1,211 @@
+/*******************************************************************************
+ * Copyright (c) 2008-2019 German Aerospace Center (DLR), Simulation and Software Technology, Germany.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package de.dlr.sc.virsat.model.extension.requirements.csv;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * A class to read a CSV file and pars it to lists
+ *
+ */
+public class CsvFileReader {

```

Comment:
I would rather use an existing CSV library to be honest

Reply:
That doesn't read the whole file twice

Reply:
https://stackoverflow.com/questions/14274259/read-csv-with-scanner/24950812#24950812

Reply:
Guess, I have to agree...

Reply:
I'm using apache commons csv parser now ;) And updated it to not read the file multiple times